accounts/abi/bind/bind.go

```diff
@@ -41,6 +41,26 @@ const (
 	LangObjC
 )
 
+type BindOpts struct {
+	Abi string
+	Bin string
+	Typ string
+
+	Link       string
+	SolFiles   string
+	RemapPaths string
+	Compiler   string
+	Exec       string
+
+	Lang       Lang
+	Pkg        string
+	OutputFile string
+}
+
+func CompileAndBind(files ...string)
+
+func BindIndividualFiles(abis string, bytecodes string, pkg string, lang Lang)
```

Comment:
Specifically these are what I'm contemplating about adding in but it might be better to do this in a separate PR and just get it compiling like it normally does first.